rocky/components/modal/template.html

```diff
@@ -0,0 +1,32 @@
+{% load static %}
+{% load i18n %}
+{% load compress %}
+
+<div class="modal-wrapper">
+    {% slot "trigger" %}
+        <button class="modal-trigger" data-target="{{ dialogid }}">Open modal {{ dialogid }}</button>
```

Comment:
dialogid needs to be html ID safe here, eg, no spaces etc. better not to rely on ID's at all.